engine/src/parameters.rs

```diff
@@ -237,6 +237,8 @@ pub struct BeginBlockArgs {
 #[derive(Debug, Clone)]
 pub struct NEP141FtOnTransferArgs {
     pub sender_id: AccountId,
+    /// Balance can be for Eth on Near and for Eth to Aurora
+    /// We use common `Balance` type without concrete specification.
```

Comment:
I think the comment here should actually say something like

"`ft_on_transfer` can be called with arbitrary NEP-141 tokens attached, therefore we do not specify a particular type Wei."

If it were true that this only ever received some kind of Wei (either `NEP141Wei` or `Wei`) then I would say that we should make this type `NEP141Wei`.

Reply:
Comment changed.